network/tests/common.py

```diff
@@ -10,6 +10,10 @@
 
 INSTANCE_BLACKLIST = {"collect_connection_state": True, "blacklist_conntrack_metrics": ["count"]}
 
+
+# In order to collect connection state we need `ss` command included in `iproute2` package
+E2E_METADATA = {'start_commands': ['apt-get update', 'apt-get install iproute2 -y',]}
```

Comment:
Looks like as expected, we now have more metrics. But the e2e was not failing before, so might be good to add some of the new metrics to `EXPECTED_METRICS` used for e2e:

Before 

```
    network (1.12.1)
    ----------------
      Instance ID: network:203b380dbea98c84 [OK]
      Configuration Source: file:/etc/datadog-agent/conf.d/network.d/network.yaml
      Total Runs: 2
      Metric Samples: Last Run: 37, Total: 74
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 15ms
```

Now

```
    network (1.12.1)
    ----------------
      Instance ID: network:203b380dbea98c84 [OK]
      Configuration Source: file:/etc/datadog-agent/conf.d/network.d/network.yaml
      Total Runs: 2
      Metric Samples: Last Run: 49, Total: 98
      Events: Last Run: 0, Total: 0
      Service Checks: Last Run: 0, Total: 0
      Average Execution Time : 30ms
```

Reply:
Added additional metrics: https://github.com/DataDog/integrations-core/pull/5261/commits/b8f8a8c7e3331968632f72a410fcf47b4ac2045c